build.zig

```diff
@@ -4,27 +4,32 @@ const Build = std.Build;
 pub fn build(b: *Build) !void {
     const optimize = b.standardOptimizeOption(.{});
     const target = b.standardTargetOptions(.{});
-
-    const options = Options{
-        .install_libs = true,
-        .from_source = true,
-    };
-    // Just to demonstrate/test linking. This is not a functional example, see the mach/gpu examples
-    // or Dawn C++ examples for functional example code.
-    const example = b.addExecutable(.{
-        .name = "dawn-example",
-        .root_source_file = .{ .path = "src/main.zig" },
-        .target = target,
-        .optimize = optimize,
-    });
-    try link(b, example, options);
-    const glfw_dep = b.dependency("mach_glfw", .{
-        .target = example.target,
-        .optimize = example.optimize,
-    });
-    example.linkLibrary(glfw_dep.artifact("mach-glfw"));
-    example.addModule("mach-glfw", glfw_dep.module("mach-glfw"));
-    b.installArtifact(example);
+    _ = optimize;
+    _ = target;
```

Comment:
this has a funny side effect: the CI pipeline relies on this example in order to build e.g. `zig-out/lib/dawn.a` which gets uploaded to github releases, so if I merge this it would create a broken binary release. :|

I think we need to do this:

1. `build.zig` in this repository needs to look exactly how you have it here.
2. There needs to be *another* copy of `build.zig`, call it `build.zig.local` - which looks how it did _before_ this change.
3. When CI runs, `mv build.zig.local build.zig` before running `zig build`, so that this chunk of code would be uncommented and it would still produce `zig-out/lib/` for the release